src/packages/searchbar/searchbar.tsx

```
@@ -153,11 +184,9 @@ export const SearchBar: FunctionComponent<
   }
 
   const clearaVal = (event: MouseEvent<HTMLDivElement>) => {
-    if (disabled || readOnly) {
-      return
-    }
+    if (disabled || readOnly) return
     setValue('')
-    onChange && onChange?.('')
+    onChange && onChange('')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)